docs/src/modules/components/DiamondSponsors.js

```diff
@@ -13,26 +14,24 @@ const NativeLink = styled('a')(({ theme }) => ({
   display: 'flex',
   alignItems: 'center',
   justifyContent: 'center',
-  borderRadius: 12,
-  border: '1px solid',
-  borderColor: (theme.vars || theme).palette.divider,
-  transition: theme.transitions.create(['color', 'border-color']),
-  boxShadow: `${alpha(theme.palette.grey[100], 0.3)} 0 -2px 0 inset`,
+  border: '3px solid transparent',
+  transition: theme.transitions.create(['background-color']),
+  '&:first-of-type': {
+    borderRadius: '11px 11px 0 0', // concentric border-radius
```

Comment:
It's because of the focus state—I'm using borders for that instead of `outline`. The reason for that is because `outline` is applied outside the element's bounding box, whereas the border is applied inside it. So, using `outline` for the focus-visible state wouldn't work well with these links being inside a container.